eng/targets/Imports.targets

```diff
@@ -286,6 +286,14 @@
     <Warning Condition="'%(RestrictedInternalsVisibleTo.Partner)' == '' and '%(RestrictedInternalsVisibleTo.Namespace)' == ''" Text="RestrictedInternalsVisibleTo items must specify the 'Partner' or 'Namespace' attribute. Target assembly: %(Identity)" />
   </Target>
 
+  <!--
+    Deletes the package built by this project from local NuGet cache (if present).
+    Allows refreshing locally built Roslyn packages consumed by another local repo.
```

Comment:
Alternative (I think a better one because it works in other repos and maybe more scenarios) is to specify VersionSuffix when packing, I usually do 

```ps1
.\build.cmd -pack /p:VersionSuffix=$(git rev-parse --short HEAD)
```

That produces a uniquely versioned package.

Reply:
You'd need to keep updating the version in the other repository with every build though.